.github/ISSUE_TEMPLATE/bug_report_form.yml

```
@@ -14,6 +14,15 @@ body:
         - label: This issue is not a duplicate
           required: true
 
+  - type: checkboxes
+    id: desktop-check
+    attributes:
+      label: Does it also happen in the desktop version?
+      description: If so, please report it in [Anki Forums](https://forums.ankiweb.net) instead
+      options:
+        - label: This bug does not occur in the latest version of Anki Desktop
```

```suggestion
        - label: This bug does not occur in the latest version of Anki (desktop version)
```
"Anki Desktop" is a kind of jargon, which may not be desirable to use in text for common users. Even if it looks somewhat clumsy for developers' eyes to use "version" twice, it might be clearer for the users.

Or just:

```suggestion
        - label: This bug does not occur in the latest version of Anki
```
Since the label text already describes it as "the desktop version", it might be sufficient to just rephrase it as "Anki" in this context.